src/blocks/gallery-carousel/save.js

```diff
@@ -130,7 +130,7 @@ const save = ( props ) => {
 	return (
 		<div aria-label={ __( `Carousel Gallery`, 'coblocks' ) } >
 			<div className={ innerClasses }>
-				<div className={ swiperClasses } data-swiper={ JSON.stringify( swiperOptions ) } id={ uuid } style={ swiperStyles } >
```

Comment:
Do we not need an ID on the carousel?

Reply:
No, the ID isn't used on our end. We initialize the carousels using the class name.